scripts/monte_carlo.py

```diff
@@ -13,19 +13,37 @@
 .. code:: yaml
 
     monte_carlo:
```

Comment:
You can safely delete the comments to the settings here, as for the other scripts

Reply:
I have removed the comments as advised.